### ISSUE TITLE
Document Custom TauDEM Channel Delineation

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#484]
@jtgilbert we need to document how custom-derived from topography channel delineation works in custom project runs for TauDEM (typically from high-res topo like LiDAR). See #447.   We'll need to figure out how we want to note and document these custom "store" runs.

**Comment by nbouwes** [comment]
Funny, I just emailed Kelly and Philip if there was paragraph I could use to describe this.  I then did a google search and found this thread

**Comment by joewheaton** [comment]
Here is a start: https://tools.riverscapes.xyz/taudem/

**Comment by nbouwes** [comment]
Yeah, I have used that for a talk previously.  I was looking for a way to describe how we delineated all the channels for kern meadows with the LiDAR. Made a huge difference for hand and other stuff 

_______
Nick Bouwes
From my phone
